app/components/action-feed/component.js

```diff
@@ -12,15 +12,57 @@ import { translationMacro as t } from 'ember-i18n';
  * ```
  * @class action-feed
  */
+
 export default Component.extend({
     store: service(),
     toast: service(),
-
+    currentUser: service(),
     classNames: ['action-feed'],
+    page: 1,
+    listLoading: false,
 
     errorMessage: t('components.action-feed.error_loading'),
 
     moreActions: computed('totalPages', 'page', function() {
         return this.get('page') < this.get('totalPages');
     }),
+
+    init() {
+        this._super(...arguments);
+        this.loadPage();
+        this.set('listLoading', true);
+        this.actionsList = [];
+    },
+
+    loadPage() {
+        const page = this.get('page');
+        this.get('currentUser.user').then(user => this.get('store').queryHasMany(user, 'actions', { page }).then(
```

Comment:
I think it'd be cleaner to chain these promises at the same level, like
```
this.get('currentUser.user')
    .then(user => store.queryHasMany(...))
    .then(this._setPageProperties.bind(this))
    .catch(this._handleLoadError.bind(this));
```
(using `.bind` vs `() =>` doesn't matter, but I like binding because you don't need to repeat the arguments that are being passed)